setupTests.js

```diff
@@ -5,16 +5,50 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+let hathorMemoryStorage = {};
```

Comment:
Shouldn't it be one regular implementation of a storage (instead of being here only for tests)?

Reply:
This is a regular implementation of the storage. It's just a in-memory storage, without persistence.

Reply:
I know. Maybe we should let this implementation available for whoever wants to use it. And I think we should rename it to `MemoryOnlyStore`.

Reply:
Ok, I created the class but did NOT move into a separate file so other people can use. Using memory storage is very dangerous and users should not do it unless they know exactly what they're doing. In that case, they can just implement their own.